src/features/lightspeed/utils/data.ts

```diff
@@ -23,6 +23,9 @@ export function shouldRequestInlineSuggestions(
     typeof parsedAnsibleDocument === "object" &&
     !Array.isArray(parsedAnsibleDocument)
   ) {
+    vscode.window.showErrorMessage(
+      "Ansible Lightspeed expects valid Ansible syntax. For playbook files it should be a list of plays and for tasks files it should be list of tasks."
```

Comment:
I'm concerned novice users will struggle with this message. Can we detect if it's a playbook at least and tailor the message better? "A playbook must be a list of plays." or "A tasks file must be a list of tasks."

Reply:
The problem is when user writes `name: Deploy web servers` and hits enter there is no certain way to identify if they are writing a playbook file or a task file (which they intend to include in another playbook).